evaluators/rank/NdcgEvaluator/README.md

```diff
@@ -1,4 +1,67 @@
 # NDCGEvaluator
 
-`NDCGEvaluator` measures the performance of a retrieval system based on the graded relevance of the predicted scores and desired scores. It ranges from `0.0` to `1.0`, with `1.0` representing the ideal ranking of the retrieved result.
- 
\ No newline at end of file
+`NDCGEvaluator` computes the [NDCG (Normalized Discounted Cumulative Gain)](https://en.wikipedia.org/wiki/Discounted_cumulative_gain). 
+It measures the performance of a retrieval system based on the graded relevance of the predicted scores and desired scores. 
+It ranges from `0.0` to `1.0`, with `1.0` representing the ideal ranking of the retrieved result.
+
+## Usage example.
+A simple example on how one would use it in Python.
+
+```python
+expected = [('1', 5.0), ('2', 3.0), ('3', 4.0)]
+results = [('2', 1.0), ('1', 0.8), ('3', 0.7)]
+evaluator = NDCGEvaluator(eval_at=5, power_relevance=False)
+evaluation = evaluator.evaluate(actual=results, desired=expected)
+```
```

Comment:
maybe we can introduce a bit on `power_relevance` here

Reply:
Okey, u can do it as part of the fix if u want

Reply:
I explained above